Include/internal/pycore_optimizer.h

```diff
@@ -31,7 +31,8 @@ typedef struct {
     uint8_t oparg;
     uint16_t valid:1;
     uint16_t linked:1;
-    uint16_t chain_depth:14;  // Must be big engough for MAX_CHAIN_DEPTH - 1.
+    bool was_run:1;
```

Comment:
While it shouldn't change on modern compilers, I think compilers like consecutive types, so I'd suggest using a `uint16_t:1` (I think the structure will have `sizeof` 64, whether you use bool or uint16_t, at least on gcc, but I'm not sure on other compilers; it would also be consistent with the types of `valid` and `linked`).